string robot_id
string reward_id
string hyperparams_id
string network_architecture_id
string user_id
string name
string task_id